interp/src/debugger/cidr.rs

```diff
@@ -96,6 +101,10 @@ impl Debugger {
         let mut ctx = ir::from_ast::ast_to_ir(ws)?;
         let pm = PassManager::default_passes()?;
 
+        // Metadata stuff
+        let metadata = ctx.metadata.clone().unwrap();
+        let mapping = parse_metadata(&metadata).unwrap();
```

Comment:
You want to use the map method of `Option` so that we don't explode in the case
where the metadata isn't present, rather than using `unwrap`